about.html

```diff
@@ -0,0 +1,225 @@
+            />
+          </div>
+        </div>
+      </section>
+      <section class="past-summits">
+        <div class="container">
+          <h2 class="past-summits__headline small-underline">
+            See the Past African Education Summit
+          </h2>
+          <p class="past-summits__paragraph">
+            Take a look at the previous two African Education Summits held in
+            Accra and Kinshasa.
+          </p>
+          <ul class="past-summits__cards-wrapper">
+            <li class="past-summits__card">
+              <img
+                class="past-summits__card__img"
+                src="assets/images/accra-2021.jpg"
+                alt="African Education Summit in Accra 2021"
+              />
+              <div class="past-summits__card__body">
+                <h4 class="past-summits__card__header">2021</h4>
+                <p class="past-summits__card__description">
+                  African Education Summit 2021 in Accra
+                </p>
+                <button class="past-summits__card__btn" type="button">
+                  See More
+                </button>
+              </div>
+            </li>
+            <li class="past-summits__card">
+              <img
+                class="past-summits__card__img"
+                src="assets/images/kinshasa-2022.jpg"
+                alt="African Education Summit in Kinshasa 2022"
+              />
+              <div class="past-summits__card__body">
+                <h4 class="past-summits__card__header">2022</h4>
+                <p class="past-summits__card__description">
+                  African Education Summit 2022 in Kinshasa
+                </p>
+                <button class="past-summits__card__btn" type="button">
+                  See More
+                </button>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <div id="overlay" class="overlay"></div>
+    </main>
+    <footer class="footer">
+      <div class="container">
+        <div class="footer__wrapper">
+          <a href="/" class="footer__logo__wrapper">
+            <img class="footer__logo" src="assets/icons/logo-lg.svg" alt="" />
+          </a>
+          <div>
+            <p class="footer__copyright">
+              &#169; 2023 African Education Summit. Some rights reserved.
+            </p>
+            <p class="footer__disclaimer">
+              All eniities represented on this website, including the partners,
+              speakers and program, are all fictional and were created for the
+              purpose of a web development project. The source code for this
+              website can be found
+              <a
+                class="footer__github-link"
+                href="https://github.com/henokkhm/microverse-module-1-capstone-project"
+              >
+                here </a
+              >.
+            </p>
+          </div>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] The footer in the about page should have a dark background, kindly make sure to implement this so your website looks identical to the provided design

|Your website|Provided design|
|-|-|
|![Screenshot_20230529_231637](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/25227022/f0ac1eb2-2fbb-4b79-bee1-039eea5feee6)|![Screenshot_20230529_231729](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/25227022/cefd9ff6-d38a-4b9e-8d6c-6483c4a36f74)|